### ISSUE TITLE
Stucco/plaster building material

### ISSUE THREAD

**Issue by hitzu111** [StrangeLoopGames/EcoSuggestions#1265]
While stucco itself is not a suitable building material itself it is commonly used to protect and decorate buildings.
In Eco, it could translate into Plastered Mortared Stone, Plastered Brick, Plastered Lumber, and even Plastered Concrete. In a basic way, it could just be used as an option for a tier upgrade by mixing one of the existing building materials with plaster/stucco.
A more sophisticated way of implementing this could go along the way with the paint suggestion #224

**Comment by legionaw** [comment]
Yes, please!!! I LOVE [Spanish Colonial architectural style](https://en.wikipedia.org/wiki/Spanish_Colonial_architecture) and its [Revival style counterpart](https://en.wikipedia.org/wiki/Spanish_Colonial_Revival_architecture). Perfect for building in both Rain Forest and Desert biomes. We really need this. +1
